modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
+
+* With this update, Repo server storage used for storing manifests can be extended by mounting Persistent Volumes. The default storage volume utilized by the Repo server is `EmptyDir`, which has limitations and may prove inadequate in scenarios involving numerous repositories, sizable repositories, or repositories containing large manifests. Additional storage can be mounted when required using the `.spec.repo.volumes` and `.spec.repo.volumeMounts` fields in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-4646[GITOPS-4646]
++
+For more information, refer to the link:https://argocd-operator.readthedocs.io/en/latest/usage/ha/repo-server/#high-availability-for-repo-server[HA for Repo Server].
+
+* With this update, users can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
+
+* With this update, users can expose Argo CD Application labels on Kubernetes events generated by Argo CD. When events are created for applications that have specific label keys defined in the `resource.includeEventLabelKeys` field within the `argocd-cm` ConfigMap, the controller associates the relevant labels with these events. This enhancement facilitates the filtering and processing of events according to application labels. To set the `resource.includeEventLabelKeys` field in a {gitops-shortname} environment, use the `.spec.extraConfig` field in the Argo CD CR. The {gitops-shortname} Operator automatically incorporates these fields from the `.spec.extraConfig` field into the `argocd-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-3233[GITOPS-3233]
++
+.Example 
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-sample # <1>
+  namespace: default # <2>
+spec:
+  extraConfig:
+    resource.includeEventLabelKeys: team,env* # <3>
+----
+<1> Specifies the name of the Argo CD instance.
+<2> Specifies the namespace where the Argo CD is installed.
+<3> Specifies the label keys that match and are added onto Application events.
++
+For more information, refer to link:https://argo-cd.readthedocs.io/en/latest/operator-manual/declarative-setup/#labels-on-application-events[Labels on Application Events].
+
+* With this update, users can develop applications by verifying commit signatures for `ApplicationSet` CDR created using Git generators. Before this update, users could develop applications without the necessity of verifying commit signatures. link:https://issues.redhat.com/browse/GITOPS-3049[GITOPS-3049]
```

Comment:
CRDs? custom resource definitions? Not sure what CDRs are, but maybe they're relevant to gitops.

Reply:
It is indeed CRDs. Sorry about the typo. I have fixed the incorrect change. Thanks.